tests/regression/roles/sonic_ntp/tasks/cleanup_tests.yaml

```diff
@@ -0,0 +1,25 @@
+- name: Delete interfaces
+  vars:
+    ansible_connection: network_cli
+  sonic_config:
+    commands: "{{ preparations_tests.delete_interfaces }}"
+  register: output
+  ignore_errors: yes
+
+- name: Delete MGMT VRF
+  vars:
+    ansible_connection: network_cli
+  sonic_config:
+    commands: "{{ preparations_tests.delete_mgmt_vrf }}"
+  register: output
+  ignore_errors: yes
+
+- name: Change the interface mode to old
```

Comment:
It would be useful to do something like this in the preparation tests, but not as a cleanup task. (Currently, it is a prerequisite manual task to configure the appropriate naming mode on the switch to match the interface mode specified for the test. Adding an automated execution block like this as a top level test "task" before running any of the requested tests would be the best way to do it.)

To do a cleanup "restore" would require keeping and making use of a persistent state to indicate that the test changed the interface naming state on the switch.

Also, it looks like this "when" clause isn't in the right place for the intended purpose. If it were intended to make execution of the command conditional, it looks like (from the Ansible documentation) it would need to be at the same indentation level as "sonic_config:", just after the "commands: ..." item.

In this case, however, I would recommend deleting this "change the interface mode ..." block.

Reply:
Removedthe portion.

Reply:
Verified.